ethkey/README.md

```diff
@@ -16,29 +16,29 @@ Ethereum keys generator.
   Copyright 2016, 2017 Parity Technologies (UK) Ltd
 
 Usage:
-    ethkey info <secret> [options]
+    ethkey info <secret-or-phrase> [options]
     ethkey generate random [options]
-    ethkey generate prefix <prefix> <iterations> [options]
-    ethkey generate brain <seed> [options]
```

Comment:
this makes it backward incompatible. I was using this option a lot for my personal accounts

Reply:
Bumped version of ethkey and ethstore and added relnotes llabel.

IMHO it's more consistent now, if you provide `<seed>` we don't really `generate` anything, we just print address info (similar to when you are providing the secret), there is a `secret` derivation in the background obviously, but I think it significantly differs from `generate random` and `generate prefix` commands.